src/webapi/events/touch.rs

```diff
@@ -0,0 +1,193 @@
+use webcore::value::Reference;
+use webcore::try_from::TryInto;
+use webapi::event::{IEvent, IUiEvent, UiEvent, Event};
+use webapi::touch::Touch;
+
+/// The `ITouchEvent` interface represents events that occur due to the user
+/// interacting with a touch device (such as a phone).
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent)
+// https://w3c.github.io/uievents/#touchevent
```

Comment:
These links are not valid. (:

In this case this link should be:

https://w3c.github.io/touch-events/#idl-def-touchevent

Please fix the other links too.

If it's not possible to link to something (e.g. I think the fields in the `TouchEvent` interface cannot be linked to?) then you can skip those.